src/Microsoft.Data/DataFrameColumn.cs

```diff
@@ -173,7 +185,7 @@ public void SetName(string newName, DataFrame dataFrame = null)
         /// </summary>
         public virtual DataFrame Description() => throw new NotImplementedException();
 
-        internal virtual DataFrameColumn GetAscendingSortIndices() => throw new NotImplementedException();
+        public virtual PrimitiveDataFrameColumn<long> GetAscendingSortIndices() => throw new NotImplementedException();
```

Comment:
Does this need to be public? Or should it be `protected`?

Reply:
I initially thought it would be a hook to override in the `sort` method. It doesn't offer much value though. I'm ok with it being `protected` .